### ISSUE TITLE
Unable to download certain videos

### ISSUE THREAD

**Issue by manchan** [alexeichhorn/YouTubeKit#40]
Hi.
The following error occurs with certain videos.
Relatively recent uploads are not possible.

- https://youtu.be/VGKrQu3ERgY?si=T7uQzeIbKec0s0eP
- https://www.youtube.com/watch?v=09VuoiCzCug

finished with error [57] Error Domain=NSPOSIXErrorDomain Code=57 "Socket is not connected" UserInfo={NSErrorFailingURLStringKey=https://youtubekit-remote.losjet.com/v1?videoID=VGKrQu3ERgY, NSErrorFailingURLKey=https://youtubekit-remote.losjet.com/v1?videoID=VGKrQu3ERgY, _NSURLErrorRelatedURLSessionTaskErrorKey=(
    "LocalWebSocketTask <5A6E716A-8ADA-40C6-A620-4C8AE9294E65>.<13>"
), _NSURLErrorFailingURLSessionTaskErrorKey=LocalWebSocketTask <5A6E716A-8ADA-40C6-A620-4C8AE9294E65>.<13>}

**Comment by alexeichhorn** [comment]
Both videos work for me. I'm calling it like this: `YouTube(videoID: "09VuoiCzCug", methods: [.remote])`.
Does it work for you when you do `methods: [.local]` (i.e. just without remote)?
Also when did you notice it started to happen?

**Comment by manchan** [comment]
I also worked on the above two.
However, it seems that if the video was uploaded immediately, it cannot be downloaded.

[URL of the site where the DL was not available]
https://youtu.be/XvyTXHEcL4g?si=qw9bYHPDEnnfEqrE
https://youtu.be/BC5vkW0mT7U?si=haCv83og4l8FMcbT

etc.

[URL of the site where the DL is now available]
https://youtu.be/GMivynf6NZ8?si=jCCx7wXgeAoHnJrs
https://youtu.be/ejGQN9jPs_k?si=OI0RRdjnCnvazOLB
etc.

"URL of the site where the DL is available.
I have described it as

I tried to DL 1.2 days ago and was not able to do so, but after some time passed or a new video in the channel was uploaded, I was able to DL the video.

This is because of the following reasons.
So, the URLs that could not be DLed may be able to be DLed in the future.
